### ISSUE TITLE
Lack of Documentation

### ISSUE THREAD

**Issue by franciscoribeiro2003** [franciscoribeiro2003/sync-obsidian-tasks#1]
Lack of Documentation (Visual documentation) for non tech users know how to setup this plugin
